src/docs/sdk/performance/ui-event-transactions.mdx

```diff
@@ -85,24 +85,24 @@ Scenario: Don't overwrite existing status of UI event transactions
     Then it keeps the status 
     And doesn't overwrite it
 
-Scenario: Same UI element with same event
-    Given an ongoing UI event transaction
-    When the user triggers the same UI event with the same type for the same
-        UI element
-    Then the SDK reschedule the idle timeout
-    And doesn't create a new transaction
-
 # If your SDK binds auto-generated transactions to the scope see binding
 # to scope
-Scenario: Same UI element with different event
-    Given an ongoing UI event transaction
-    When the user triggers the same UI element with a different event
+Scenario: New event on same UI element
```

Comment:
`m`: We have now two scenarios with the same name. I would rename this to 
```suggestion
Scenario: Event on same UI element with child spans
```
and rename the scenario below to
```
Scenario: Event on same UI element without child spans
```